wiki/Guides/How_To_Get_Your_Map_Ranked/en.md

```diff
@@ -1,25 +1,22 @@
+Once your map has been pending for a while, has 5 people on your hype train, and you think it's perfect and ready to be ranked, it's time to get a [Beatmap Nominator](/wiki/People/Beatmap_Nomination_Group). Make sure you check the team member's signature or userpage to see whether they have a modding policy; if they're open to requests or there's nothing there, ask them politely in game if they wouldn't mind taking a look at your map. Lots of BN are extremely busy — there are an awful lot of pending maps to get through, so don't be too disheartened if you are turned down. Having plenty of mods and hype on your map will help encourage Beatmap Nominators to look at your map, so if you're having difficulties getting a Beatmap Nominator to look at your map, remember that your hype train and mods will help you out.
 
 ## Step 3: Bubbles
 
-Just because your map is bubbled doesn't mean your work is over! Don't ignore any mods by normal users on your bubbled map — although only a BN or QAT can rank your map, anyone can still help improve it, and a better, more thoroughly modded map means an easier job for the BNG who eventually looks at it. Once you have applied a mod on your bubble the BN who originally gave the bubble will almost always be happy to come back and reinstate it. Keep in mind that you'll also have an easier time getting your bubble noticed by BNG members if it is high priority and on the first page of pending - giving your map some kudos stars to push it up in the list should make the wait a little shorter.
+Just because your map is bubbled doesn't mean your work is done! Don't ignore mods by normal users on your bubbled map. Although only a Beatmap Nominator or Quality Assurance Team member can rank your map, anyone can help improve it. A more thoroughly modded map means an easier job for the BNG who looks at it. Once you have applied a mod on your bubble, the Beatmap Nominator who originally gave the bubble will always be happy to come back and reinstate it. Keep in mind that you'll also have an easier time getting your bubble noticed by BNG members if it's high priority and on the first page of pending. Having people build your hype train to push it up the list should make the wait for ranking shorter.
```

Comment:
some outdated terminology
- bubble => nomination
- QAT => NAT
- BNG => BN or nominator

the note about being on the first page of pending probably doesn't apply enough to mention here

Reply:
Fixed with new commits.